include/matx/operators/dct.h

```diff
@@ -102,7 +102,7 @@ void dct(OutputTensor &out, const InputTensor &in,
 
   tensor_t<cuda::std::complex<typename OutputTensor::scalar_type>, 1> tmp{{N + 1}};
 
-  fft(tmp, in, 0, stream);
+  fft_impl(tmp, in, 0, stream);
```

Comment:
do we need to rename or can we just overload?

Reply:
We talked about this on slack, but renaming is intended to be clear what's an implementation detail vs user API